plugins/woocommerce/src/Internal/DataStores/Orders/DataSynchronizer.php

```diff
@@ -335,6 +334,30 @@ public function get_background_sync_interval(): int {
 		return $interval;
 	}
 
+	/**
+	 * Keys that can be ignored during synchronization or verification.
+	 *
+	 * @since 8.6.0
+	 *
+	 * @return string[]
+	 */
+	public function get_ignored_order_props() {
+		/**
+		 * Allows modifying the list of order properties that are ignored during HPOS synchronization or verification.
+		 *
+		 * @param string[] List of order properties or meta keys.
+		 * @since 8.6.0
+		 */
+		return apply_filters(
+			'woocommerce_hpos_sync_ignored_order_props',
+			array(
+				'_paid_date', // This has been deprecated and replaced by '_date_paid' in the CPT datastore.
+				'_completed_date', // This has been deprecated and replaced by '_date_completed' in the CPT datastore.
+				'_edit_lock',
```

Comment:
Or perhaps get the `$ephemeral_meta_keys` array from `OrdersTableDataStore` and merge it with this array, so that new ephemeral meta keys only need to be added in one place?

Reply:
> Should this be `EditLock::META_KEY_NAME`?

Fixed in f903add6c0145adfdb77690bc9bba1fc202bc442.

> so that new ephemeral meta keys only need to be added in one place?

I guess we could do that, though I wanted to keep both sets of keys separate for now as (in theory) they serve different purposes and maybe there are keys that must not trigger a full order save but we do want to sync during a migration (like maybe expensive to compute caches or something like that).